renku_notebooks/api/notebooks.py

```diff
@@ -337,6 +360,20 @@ def stop_server(user, forced, server_name):
         return make_response(jsonify({"messages": {"error": "Server not found."}}), 404)
 
     r = delete_named_server(user, server_name)
+
+    # If the server was deleted gracefully, remove the PVC if it exists
+    if r.status_code < 300:
```

Comment:
Would it make sense to allow the user to decide between "stop" and "hibernate"? In the second case, the PVS is kept when gracefully deleting.

Reply:
Hmm maybe - but I would see that as an extension of the present functionality. There was also some discussion to potentially provide long-lived volumes for users in general.